src/game/states/play.rs

```diff
@@ -456,7 +461,7 @@ impl GameState for Play {
         let img = s.assets.get_img(ctx, "common/crosshair");
         graphics::draw(ctx, &*img, drawparams)
     }
-    fn event_up(&mut self, s: &mut State, ctx: &mut Context, event: Event) {
+    fn event_down(&mut self, s: &mut State, ctx: &mut Context, event: Event) {
```

Comment:
This changes the behaviour the of everything else as well. Is this really what we want?